### PR TITLE
[BpkSwapButton]: Updating exported typescript declarations for docs and usage

### PR DESCRIPTION
**Pull request by olliecurtis**
Updates the TS declaration files for exports to be reflective of current API, also exports `SWAPBUTTON_TYPES` so it can be used outside of the component for types

Remember to include the following changes:

- [ ] Ensure the PR title includes the name of the component you are changing so it's clear in the release notes for consumers of the changes in the version e.g `[KOA-123][BpkButton] Updating the colour`
- [ ] `README.md` (If you have created a new component)
- [x] Component `README.md`
- [x] Tests
- [x] Storybook examples created/updated
- [ ] For breaking changes or deprecating components/properties, migration guides added to the description of the PR. If the guide has large changes, consider creating a new Markdown page inside the component's docs folder and link it here

### CODE REVIEW

File: packages/bpk-component-swap-button/src/BpkSwapButton.d.ts [hunk 1]
```diff
@@ -16,14 +16,20 @@
...
 
+export declare const SWAPBUTTON_STYLES: {
+    surfaceContrast: string;
+    canvasDefault: string;
+    canvasContrast: string;
+};
+export type SwapButtonStyle = (typeof SWAPBUTTON_STYLES)[keyof typeof SWAPBUTTON_STYLES];
 export type Props = {
-  ariaLabel: string;
-  ariaLiveTextProps: string;
```
Comment (arnaugm) [inline packages/bpk-component-swap-button/src/BpkSwapButton.d.ts:21]: this type disappears, is intentional?

File: packages/bpk-component-swap-button/src/BpkSwapButton.d.ts [hunk 2]
```diff
@@ -16,14 +16,20 @@
...
-  onClick: () => void;
+    onClick: () => void;
+    ariaLabel: string;
+    swapButtonStyle?: SwapButtonStyle;
 };
 declare const BpkSwapButton: ({
-  ariaLabel,
-  ariaLiveTextProps,
```
Comment (arnaugm) [inline packages/bpk-component-swap-button/src/BpkSwapButton.d.ts:26]: same here

File: packages/bpk-component-swap-button/src/BpkSwapButton.d.ts (hunk 1 above)
Comment (olliecurtis) [reply packages/bpk-component-swap-button/src/BpkSwapButton.d.ts:21]: Yes this was removed in a previous PR from Luna as it is handled on the consumer side but I missed in my review: https://github.com/Skyscanner/backpack/pull/3197

File: packages/bpk-component-swap-button/src/BpkSwapButton.d.ts (hunk 2 above)
Comment (olliecurtis) [reply packages/bpk-component-swap-button/src/BpkSwapButton.d.ts:26]: See above

**Review by arnaugm — APPROVED** [review]

**Pull request merged**